components/Menu/Menu.js

```diff
@@ -80,6 +87,9 @@ export default function Menu({
             {isCollections && (
               <StyledSubNavigationList>
                 {collections.map((collection) => {
+                  const numberOfCards = getNumberOfCardsInCollection(
```

Comment:
```js
{collections.map((collection) => {
  return (
    <StyledSubNavigationListItem
      key={collection.id}
      $isCollectionsClosing={isCollectionsClosing}
    >
      <StyledSubNavigationLink
        href={`/${collection.id}`}
        onClick={() => {
          changeFlashcardSelection("all");
          setIsMenuClosing(true);
          setIsCollectionsClosing(true);
        }}
      >
        {collection.title} ({getAllFlashcardsFromCollection(collection.id).length})
      </StyledSubNavigationLink>
    </StyledSubNavigationListItem>
  );
})}
```

Reply:
Your version replaces eight lines of mine with half a line of yours. 😁  You're right, of course, it's much shorter this way. I sometimes find it easier to understand to take the detour via an extra function, but here I've adopted your suggestion. :+1: